### ISSUE TITLE
Wrong BOM part list

### ISSUE THREAD

**Issue by Eisbaeeer** [SasaKaranovic/DIY-Camera-Slider#2]
Hi Sascha
https://github.com/SasaKaranovic/DIY-Camera-Slider/tree/master/Hardware#part-list

Wrong:
1x [6mm GT2 280mm Closed Loop Timing Belt]
Correct:
1x[6mm GT2 200mm Closed Loop Timing Belt]

Wrong:
2x [GT2 Idler 6mm]
Correct:
1x [GT2 Idler 6mm]

**Comment by ghtomcat** [comment]
Also, it would help to link where to buy the endswitch. And which bore the idler has (3mm or 5mm). Thank you for your great project!

**Comment by Eisbaeeer** [comment]
> And which bore the idler has (3mm or 5mm).
The idler in project has 3mm. I used standard 5mm and drilled up from 3 to 5.
You have to set the right hight with spacer.
Endswitches are the small one from e.g. Omron like D2FC.
Hope that helped!

**Comment by ghtomcat** [comment]
@Eisbaeeer thanks for your help! I printed all the 3D Parts and now i'm waiting for the other parts to arrive

**Comment by SasaKaranovic** [comment]
Thanks @Eisbaeeer !
The 280mm was used in the "old" version that uses 2040 aluminum extrusion instead of 2020. Good catch, changed it to 200mm.
And you are right, it's just one GT2 pulley.

@ghtomcat I believe the pulley I had laying around was 5mm bore and since I'm using 3mm screw I had to 3D print a tiny "sleeve" to keep pulley centered. If 3mm screw fits trough the 3mm pulley then I would say go with that.
Good catch! I have updated the BoM to include end-switch. Also I would love to see how your camera slider turns out!
